### DISCUSSION TITLE
Neat way to plot allele lineage count over time?

### DISCUSSION THREAD

**Discussion by hyanwong** [tskit-dev/tskit#2500] [Q&A]
it would be nice if we could show how to calculate the number of lineages carrying a specific derived allele over time (or perhaps, the descendants of a particular mutation over time). In the case of a constant population size, this would be equivalent to the allele frequency over time, which is a common popgen thing to plot out. In the case of a fixed allele, this is simply the number of lineages over time, of course. Is there a neat way to do this?

It would also be interesting to look at the rest of the ancestral haplotype containing that mutation, and on the same plot show the trajectory of those pieces of haplotype over time. There could be a nice incremental way to do this, perhaps? When interpreting, you would need to be wary of ascertainment bias, because we can't see any of the haplotype regions that are lost, but I think the idea is a nice one.

**Comment by petrelharp** [comment]
Related: https://github.com/tskit-dev/tskit/issues/386
